src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.html

```diff
@@ -267,142 +311,166 @@ <h3>
-                </td>
-                <td>
-                    <span>{{ 'artemisApp.examManagement.checklist.descriptionItem.assessAllSubmissions' | artemisTranslate }}</span>
+        </tbody>
+    </table>
+    <hr />
+    <ng-container *ngIf="!isTestExam">
+        <h3>{{ 'artemisApp.examStatus.correction.examCorrection' | artemisTranslate }}</h3>
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>
+                        <span>#</span>
+                    </th>
+                    <th>
+                        <span>{{ 'artemisApp.examManagement.checklist.checklistItem' | artemisTranslate }}</span>
+                    </th>
+                    <th>
+                        <span>{{ 'artemisApp.examManagement.checklist.description' | artemisTranslate }}</span>
+                    </th>
+                    <th>
+                        <span>{{ 'artemisApp.examManagement.checklist.goTo' | artemisTranslate }}</span>
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>1</td>
+                    <td>
+                        <span>{{ 'artemisApp.examManagement.checklist.tableItem.assessAllSubmissions' | artemisTranslate }}</span>
+                    </td>
+                    <td>
+                        <span>{{ 'artemisApp.examManagement.checklist.descriptionItem.assessAllSubmissions' | artemisTranslate }}</span>
 
-                    <hr />
-                    <!-- Display assessment progress summed up over all exercises, by correction round -->
+                        <hr />
+                        <!-- Display assessment progress summed up over all exercises, by correction round -->
 
-                    <div *ngIf="examChecklist && examChecklist.numberOfTotalExamAssessmentsFinishedByCorrectionRound !== null && examChecklist.numberOfExamsSubmitted !== 0">
-                        <div *ngFor="let finishedByCorrectionRound of examChecklist.numberOfTotalExamAssessmentsFinishedByCorrectionRound; let index = index">
-                            <hr *ngIf="index > 0" />
-                            <span> {{ 'artemisApp.examManagement.checklist.textitems.correctionRound' | artemisTranslate }} {{ index + 1 }}: </span>
+                        <div *ngIf="examChecklist && examChecklist.numberOfTotalExamAssessmentsFinishedByCorrectionRound !== null && examChecklist.numberOfExamsSubmitted !== 0">
+                            <div *ngFor="let finishedByCorrectionRound of examChecklist.numberOfTotalExamAssessmentsFinishedByCorrectionRound; let index = index">
+                                <hr *ngIf="index > 0" />
+                                <span> {{ 'artemisApp.examManagement.checklist.textitems.correctionRound' | artemisTranslate }} {{ index + 1 }} : </span>
+                                <div style="padding-top: 5px">
```

Comment:
Better use bootstrap classes or define styles in the .scss.

Reply:
This part was originally not changed by me, maybe this was marked as new when I inserted the headers for the different exam states. 
However, I have changed it to `pt-2`